R/aaa_utils.R

```diff
@@ -332,7 +359,7 @@ get_slide_id = function(x) {
 
 #' @export
 #' @rdname get_slide_id
-make_slide_url = function(x) {
+make_slide_url <- function(x) {
```

Comment:
You may want to consider (probably in a different PR so as to keep this focused on what you've done already) running this through styler so it styles the code for you. https://styler.r-lib.org/articles/introducing_styler.html

Reply:
That's a great idea. I'll run this through styler in a separate PR.